content/tests/components/common/HideFromTests.test.tsx

```diff
@@ -3,12 +3,32 @@ import React from "react";
 
 import { HideFromTests } from "devtools/components/common/HideFromTests";
 
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace NodeJS {
+    interface Global {
+      __TESTING__: boolean;
+    }
+  }
+}
+
 describe("HideFromTests", () => {
   afterEach(async () => {
+    global.__TESTING__ = true;
     await cleanup();
   });
 
-  it("should hide it's children", async () => {
+  it("show not hide it's children when testing", () => {
```

Comment:
Should this be "should not hide its children when not testing"?

Reply:
gah!